lib/std/Build/Step/InstallDir.zig

```diff
@@ -78,6 +83,16 @@ fn make(step: *Step, prog_node: *std.Progress.Node) !void {
                 continue :next_entry;
             }
         }
+        if (self.options.include_extensions) |incs| {
+            var found = false;
+            for (incs) |inc| {
+                if (mem.endsWith(u8, entry.path, inc)) {
+                    found = true;
+                    break;
+                }
+            }
+            if (!found) continue :next_entry;
```

Comment:
No need to use a mutable temporary:

```suggestion
            for (incs) |inc| {
                if (mem.endsWith(u8, entry.path, inc)) break;
            } else continue :next_entry;
```